Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md

```diff
@@ -13,17 +9,30 @@ You need to grant Cortex XSOAR authorization to access Azure Active Directory Us
 
 ## Authorize Cortex XSOAR for Azure Active Directory Users (self-deployed configuration)
 
-Follow these steps for a self-deployed configuration.
+There are two different authentication methods for self-deployed configuration: 
+- [client credentials flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow)
+- [authorization code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow).
+
+We recommend using the client credentials flow.
+
+In order to use the ***msgraph-user-change-password*** command, you must configure with the authorization code flow.
+
+Note: When using the authorization code flow, make sure the user you authenticate with has the correct roles in Azure AD in order to use the command.
 
-1. To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. To add the registration, refer to the following [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
-2. Make sure the following permissions are granted for the app registration:
-   -  API/Permission name `Directory.AccessAsUser.All` of type `Delegated`
-3. Copy the following URL and replace the ***TENANT_ID***, ***CLIENT_ID*** and ***REDIRECT_URI*** with your own client ID and redirect URI, accordingly.
+### Self deployed configuration with client credentials flow
```

Comment:
this and the below section are already documented in our ms auth in xsoar.pan.dev, lets just refer to there